app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -51,6 +50,11 @@ function DocumentCaptureReviewIssues({
         isFailedDocType={isFailedDocType}
         altFailedDocTypeMsg={isFailedDocType ? t('doc_auth.errors.doc.wrong_id_type') : null}
       />
+      {!isFailedDocType && (
+        <a href="https://www.login.gov/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/">
```

Comment:
Can we use the `getHelpCenterURL` helper from `MarketingSite` context to avoid hardcoding this link / preserve the user's preferred language?

https://github.com/18F/identity-idp/blob/main/app/javascript/packages/document-capture/context/marketing-site.tsx